packages/components/molecules/f-mega-modal/CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v7.4.0
+------------------------------
+*April 12, 2023*
+
+### Added
+- Prop `closeButtonStyle` allows either a chevron icon `chevron-left-icon` or close icon `close-small-icon` to be used.
+- `isModeRightToLeft` prop to allow a chevron icon to be used in a right to left tenant.
```

Comment:
I agree with Jamie that it'd be better to look at doing this with logical values rather than positional values eventually, rather than setting RTL as a mode.

It's outside the scope of this ticket to do that though – could we create an EE task that could look at converting the modal over to RTL using logical values potentially? (as I think it'll need a bit of a revamp in the styling to do this)

Reply:
Yup i think it's a good idea. I was thinking maybe for the WC version if we're planning on porting this over there?